tgstation.dme

```diff
@@ -313,8 +313,8 @@
 #include "code\game\gamemodes\changeling\powers\mimic_voice.dm"
 #include "code\game\gamemodes\changeling\powers\mutations.dm"
 #include "code\game\gamemodes\changeling\powers\panacea.dm"
-#include "code\game\gamemodes\changeling\powers\revive.dm"
 #include "code\game\gamemodes\changeling\powers\regenerate.dm"
+#include "code\game\gamemodes\changeling\powers\revive.dm"

```

Comment:
I'm pretty sure this is coiax's fault for not actually using DM to code; it auto-applies this as a change if you ARE using DM to code, because sorting or something